src/electron/tray.main.ts

```diff
@@ -121,14 +121,17 @@ export class TrayMain {
     }
 
     private toggleWindow() {
-        if (this.windowMain.win == null) {
-            return;
-        }
-
-        if (this.windowMain.win.isVisible()) {
-            this.windowMain.win.hide();
+        if (this.windowMain.win === null) {
```

Comment:
Should we make this condition only for macos then?

Reply:
Will do, thanks for the feedback. :)